src/components/Main/Results/DeterministicLinePlot.tsx

```diff
@@ -32,7 +32,7 @@ interface LineProps {
   key: string,
   name: string,
   color: string
-  legendType?: string
+  legendType?: import('recharts').LineProps['legendType']
```

Comment:
I am surprised this works at all. Technically, dynamic `import()` returns a promise, if I remember correctly.
Could we do a proper static `import` in the file header instead?
P.S. I don't remember if we have `type-only` import supported in our config, that would be an even better option.